radicle-cob/src/history/entry.rs

```diff
@@ -48,8 +48,6 @@ pub struct Entry {
     pub(super) id: EntryId,
     /// The actor that authored this entry.
     pub(super) actor: PublicKey,
-    /// The content-address for the resource this entry lives under.
-    pub(super) resource: Oid,
```

Comment:
I saw in the commit message you say this is redundant, why is that?

Reply:
Ahh, actually this is a mistake! I thought it was the repo id, which never changes, but it's actually the latest document oid. I'll revert and add a bit more info to the docstring.